components/feature/sitepermissions/src/main/res/values/strings.xml

```diff
@@ -1,43 +1,33 @@
-<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <resources>
-
     <!-- Title of a dialog for a notification permission request. -->
     <string name="mozac_feature_sitepermissions_notification_title">Allow %1$s to send notifications?</string>
-
     <!-- Title of a dialog for a camera permission request. -->
     <string name="mozac_feature_sitepermissions_camera_title">Allow %1$s to use your camera?</string>
-
     <!-- Title of a dialog for a microphone permission request. -->
     <string name="mozac_feature_sitepermissions_microfone_title">Allow %1$s to use your microphone?</string>
-
     <!-- Title of a dialog for a location permission request. -->
     <string name="mozac_feature_sitepermissions_location_title">Allow %1$s to use your location?</string>
-
     <!-- Title of a dialog for a camera and microphone permission request. -->
     <string name="mozac_feature_sitepermissions_camera_and_microphone">Allow %1$s to use your camera and microphone?</string>
-
     <!-- Option in a dialog for requesting a microphone permission, this option will give access to
          the first microphone-->
     <string name="mozac_feature_sitepermissions_option_microphone_one">Microphone 1</string>
-
     <!-- Option in a dialog for requesting a camera permission, this option will give access to
          back facing camera-->
-    <string name="mozac_feature_sitepermissions_back_facing_camera">Back facing camera</string>
-
+    <string name="mozac_feature_sitepermissions_back_facing_camera2">Rear-facing camera</string>
     <!-- Option in a dialog for requesting a camera permission, this option will give access to
      front facing camera-->
-    <string name="mozac_feature_sitepermissions_selfie_camera">Selfie back camera</string>
-
+    <string name="mozac_feature_sitepermissions_selfie_camera2">Front-facing camera</string>
```

Comment:
Some of the strings values from site permissions have been updated to more like Fennec style.  I took them from the Fenix mock-ups, now we are changing them, are we going to provide a way for apps to customize them or overridden the strings on the apps will be the preferred alternative?   As we are changing the strings ids the last alternative could be fragile.

Reply:
Those strings are superseding the ones from the mocks. Fenix is supposed to use those updated strings.

Reply:
👍